### DISCUSSION TITLE
New type of buttons

### DISCUSSION THREAD

**Discussion by nisarga-kh** [sButtons/sbuttons#1541] [Ideas]
Hello!! 
I read about sButtons and the amazing work they are doing. I am interested in contributing to sButtons with one new type of button. I have read all the community guidelines and I am looking forward to do some work. 
Please do let me know if I can contribute.

**Comment by shahednasser** [comment]
Hello @nisarga-kh of course you can contribute! What kind of button are you thinking of?

**Reply by nisarga-kh** [reply]
@shahednasser Thank you for responding. I am thinking of creating an animation button.

**Reply by shahednasser** [reply]
once you have a clear idea of the button you would like to add, please submit a "New button idea" issue and add the details there.
